hivemind/client/averaging/__init__.py

```diff
@@ -20,23 +20,22 @@
 import numpy as np
 
 from hivemind.dht import DHT, DHTID
-from hivemind.client.averaging.allreduce import AllReduceRunner, AllreduceException, GroupID, AveragingMode
+from hivemind.client.averaging.allreduce import AllReduceProtocol, AllreduceException, GroupID, AveragingMode
```

Comment:
Before:
- AllreduceProtocol contains all logic (splitting, averaging)
- AllreduceRunner contains protocol implementation

After:
- TensorPartContainer contains splitting logic
- TensorPartAverager contains averaging logic
- AllReduceProtocol contains protocol implementation

Reply:
renamed everything to AllReduceRunner